### PR TITLE
feat: turn DYNAMIC_LP_ASSETS flag off

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

tl;dr UNI-V2 subgraph is borked, meaning Zapper return us no UNI-V2 opportunities, meaning UNI-V2 LPs and ETH/FOX staking are borked. This makes them great again until the subgraph is fixed.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Conceptually none, we still have the old implementation's branching. Potentially will be spamming our JSON-RPC nodes in case users have cached data since it will fetch 600+ pools data from the contracts.

## Testing

- Clear your persisted store
- Ensure ETH/FOX LP and staking are happy

### Engineering

- ☝🏽 

### Operations

- ☝🏽

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4800** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4800" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4800?utm_source=stack-comment).
